pkg/virt-controller/watch/application.go

```diff
@@ -254,11 +255,21 @@ type VirtControllerApp struct {
 var _ service.Service = &VirtControllerApp{}
 
 func init() {
-	vsv1.AddToScheme(scheme.Scheme)
-	snapshotv1.AddToScheme(scheme.Scheme)
-	exportv1.AddToScheme(scheme.Scheme)
-	poolv1.AddToScheme(scheme.Scheme)
-	clonev1alpha1.AddToScheme(scheme.Scheme)
+	if err := vsv1.AddToScheme(scheme.Scheme); err != nil {
+		panic(fmt.Sprintf("error registering volumesnapshot CRDs: %v", err))
+	}
+	if err := snapshotv1.AddToScheme(scheme.Scheme); err != nil {
+		panic(fmt.Sprintf("error registering snapshot CRDs: %v", err))
+	}
+	if err := exportv1.AddToScheme(scheme.Scheme); err != nil {
+		panic(fmt.Sprintf("error registering export CRDs: %v", err))
+	}
+	if err := poolv1.AddToScheme(scheme.Scheme); err != nil {
+		panic(fmt.Sprintf("error registering pool CRDs: %v", err))
+	}
+	if err := clonev1alpha1.AddToScheme(scheme.Scheme); err != nil {
+		panic(fmt.Sprintf("error registering clone CRDs: %v", err))
+	}
```

Comment:
You can use `utilruntime.Must()` instead of `if` statements and panic. Like it is done in the generated code:
https://github.com/kubevirt/kubevirt/blob/4eb29b83bc52bfd25bd46922af8a8269f2b21913/staging/src/kubevirt.io/client-go/generated/kubevirt/clientset/versioned/scheme/register.go#L65

Reply:
Done.